<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CScope</class>
 <widget class="QDialog" name="CScope">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>346</width>
    <height>186</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CScope Lookup</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="typeLabel">
       <property name="text">
        <string>Type</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="typeComboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>260</width>
         <height>0</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>References To</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Definition Of</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Functions Called By</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Functions Calling</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Find Text</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>?</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>-</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Find File</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Including Files</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="symbolLabel">
       <property name="text">
        <string>Symbol</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="symbolString">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>260</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="subStringCheckBox">
     <property name="text">
      <string>Search for a Sub-String</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="caseInsensitiveCheckBox">
     <property name="text">
      <string>Case Insensitive</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="executeButton">
       <property name="text">
        <string>Search</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
